Week 4/MyBookCollection/app/src/main/java/com/carol/mybookcollection/MainActivity.kt

```diff
@@ -21,29 +21,31 @@ class MainActivity : AppCompatActivity() {
         setContentView(R.layout.activity_main)
```

Comment:
Let me start by saying you have excellent taste in books. I believe the book class could be a data class and then not have to have the constructor, but the way you've written it does work and you've created a nice list of books.